docs/conceptual/iommu.rst

```diff
@@ -0,0 +1,55 @@
+.. meta::
+   :description: Input-Output Memory Management Unit (IOMMU)
+   :keywords: IOMMU, DMA, PCIe, xGMI, AMD, ROCm
+
+****************************************************************
+Input-Output Memory Management Unit (IOMMU)
+****************************************************************
+
+The I/O Memory Management Unit (IOMMU) provides memory remapping services for I/O devices. It adds support for address translation and system memory access protection on direct memory access (DMA) transfers from peripheral devices. 
+
+The IOMMU's memory remapping services are used to:
+
+* provide private I/O space for devices used in a guest virtual machine.
+* prevent unauthorized DMA requests to system memory and to memory-mapped I/O (MMIO).
+* help in debugging memory access issues.
+* facilitate peer-to-peer DMA.
+
+The IOMMU also provides interrupt remapping, which is used by devices that support multiple interrupts and for interrupt delivery on hardware platforms with a large number of cores.
+
+.. note::
+
+  AMD Instinct accelerators are connected via XGMI links and don't use PCI/PCIe for peer-to-peer DMA. Because PCI/PCIe is not used for peer-to-peer DMA, there are no device physical addressing limitations or platform root port limitations. However, because non-GPU devices such as RDMA NICs use PCIe for peer-to-peer DMA, there might still be physical addressing limitations and platform root port limitations when these non-GPU devices interact with other devices, including GPUs.
```

Comment:
```suggestion
  AMD Instinct accelerators are connected via XGMI links and don't use PCI/PCIe for peer-to-peer DMA. Because PCI/PCIe is not used for peer-to-peer DMA, there are no device physical addressing limitations or platform root port limitations. However, because non-GPU devices such as RDMA NICs use PCIe for peer-to-peer DMA, there might still be physical addressing and platform root port limitations when these non-GPU devices interact with other devices, including GPUs.
```